.gitignore

```diff
@@ -23,7 +23,7 @@ bld/
 build/
 build.stm32/
 build.mingw/
-cmake-build-debug/
+cmake-build-*/
```

Comment:
Not a comment on the actual change, but I've just noticed that the previous three lines are duplicated with line 340/341... (except that one has a wildcard)